COMETwebapp/Components/PublishedValues.razor

```diff
@@ -41,42 +42,158 @@ Copyright (c) 2022 RHEA System S.A.
     /// </summary>
     List<DataChart> Data = new List<DataChart>();
 
+    /// <summary>
+    /// Option filter for the data 
+    /// </summary>
+    [Parameter]
+    public string? FilterOption { get; set; }
+
+    /// <summary>
+    /// State filter for the data 
+    /// </summary>
+    [Parameter]
+    public string? FilterState { get; set; }
```

Comment:
can you be more specific what information of the `State` is being used? can we assume you are using the iid?